Updating for Rosen Activities

cannonQ opened the pull request:
1.  Did not remove any addresses or tokens listed prior. 
2.  Added wallets for collateral locked in bridge and associated tokens 
3.  Added wallets for permits/staking as these permits were migrated to new wallets and added BTC/ETH watcher permit address 
4.  Added //notation to each address & token so it is a useful file to audit and adjust in the future 
5.  Added ETH hot & cold wallet in //notation but did not add to tvl as I'm not familiar with adding new chains to the TVL file.

 _** Open/Follow-up would be adding ETH for the TVL file, addresses are noted in this file._

Reference page where the tokens and hot/cold wallets are found is:  
- https://app.rosen.tech/assets
- https://ergexplorer.com/addressbook#offset=0&type=service&order=nameDesc

**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):


##### Twitter Link:


##### List of audit links if any:


##### Website Link:


##### Logo (High resolution, will be shown with rounded borders):


##### Current TVL:


##### Treasury Addresses (if the protocol has treasury)


##### Chain:


##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)


##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)


##### Short Description (to be shown on DefiLlama):


##### Token address and ticker if any:


##### Category (full list at https://defillama.com/categories) *Please choose only one:


##### Oracle Provider(s): Specify the oracle(s) used (e.g., Chainlink, Band, API3, TWAP, etc.):
##### Implementation Details: Briefly describe how the oracle is integrated into your project:
##### Documentation/Proof: Provide links to documentation or any other resources that verify the oracle's usage:

##### forkedFrom (Does your project originate from another project):


##### methodology (what is being counted as tvl, how is tvl being calculated):


##### Github org/user (Optional, if your code is open source, we can track activity):

llamabutler commented:
The adapter at projects/rosen-bridge exports TVL: 
        
 
 
    ergo                      12.24 M
    ergo-staking              1.27 M
    staking                   1.27 M
    cardano                   812.18 k
    bitcoin                   210.00 k
    
    total                    13.26 M

cannonQ commented:
1.  Did not remove any addresses or tokens listed prior. 
2.  Added wallets for collateral locked in bridge and associated tokens 
3.  Added wallets for permits/staking as these permits were migrated to new wallets and added BTC/ETH watcher permit address 
4.  Added //notation to each address & token so it is a useful file to audit and adjust in the future 
5.  Added ETH hot & cold wallet in //notation but did not add to tvl as I'm not familiar with adding new chains to the TVL file.

 _** Open/Follow-up would be adding ETH for the TVL file, addresses are noted in this file._

Reference page where the tokens and hot/cold wallets are found is:  
- https://app.rosen.tech/assets
- https://ergexplorer.com/addressbook#offset=0&type=service&order=nameDesc

llamabutler commented:
The adapter at projects/rosen-bridge exports TVL: 
        
 
 
    ergo                      7.24 M
    ergo-staking              1.26 M
    staking                   1.26 M
    cardano                   726.87 k
    bitcoin                   209.15 k
    ethereum                  4.28 k
    
    total                    8.18 M

g1nt0ki commented:
@cannonQ thanks for the PR, excluded RSN as it is projects own token

cannonQ commented:
@g1nt0ki There are still adjustments to be processed. I'll send a thorough comment later today.
Cheers,
CannonQ
